README.md

```diff
@@ -93,6 +93,26 @@ key file(s) (eg: `credentials_path: config/openai_api_key.txt`)
     quay.io/openshift/lightspeed-api-service-rag:latest
     ```
 
+### Optionally run inside an OpenShift environment
+In the `examples` folder is a set of YAML manifests,
+`openshift-lightspeed.yaml`. This includes all the resources necessary to get
+OpenShift Lightspeed running in a cluster. It is configured expecting to only
+use OpenAI as the inference endpoint, but you can easily modify these manifests,
+looking at the `olsconfig.yaml` to see how to alter it to work with BAM as the
+provider.
+
+Note that this creates an OpenShift Route with TLS Edge termination enabled by 
+default. Depending on your OpenShift environment, this may result in a
+self-signed certificate being associated with the endpoint, requiring you to
+change your use of curl/Postman/etc.
```

Comment:
instead of defining the route, let's just give them the "oc expose" command they can run to expose OLS via a route if they want to.